### ISSUE TITLE
mhnserver install

### ISSUE THREAD

**Issue by X4mp** [threatstream/mhn#351]
Hi, 
I used to Vagrant tutorial to setup the server and the honeypot.

Now I want to install the install_mhnserver.sh script and get the following error:

```
Initializing database, please be patient. This can take several minutes
+ python initdatabase.py
Traceback (most recent call last):
  File "initdatabase.py", line 5, in <module>
    create_clean_db()
  File "/opt/mhn/server/mhn/__init__.py", line 115, in create_clean_db
    db.create_all()
  File "/opt/mhn/env/lib/python2.7/site-packages/flask_sqlalchemy/__init__.py", line 895, in create_all
    self._execute_for_all_tables(app, bind, 'create_all')
  File "/opt/mhn/env/lib/python2.7/site-packages/flask_sqlalchemy/__init__.py", line 887, in _execute_for_all_tables
    op(bind=self.get_engine(app, bind), **extra)
  File "/opt/mhn/env/lib/python2.7/site-packages/flask_sqlalchemy/__init__.py", line 833, in get_engine
    return connector.get_engine()
  File "/opt/mhn/env/lib/python2.7/site-packages/flask_sqlalchemy/__init__.py", line 499, in get_engine
    self._engine = rv = sqlalchemy.create_engine(info, **options)
  File "/opt/mhn/env/lib/python2.7/site-packages/sqlalchemy/engine/__init__.py", line 387, in create_engine
    return strategy.create(*args, **kwargs)
  File "/opt/mhn/env/lib/python2.7/site-packages/sqlalchemy/engine/strategies.py", line 80, in create
    dbapi = dialect_cls.dbapi(**dbapi_args)
  File "/opt/mhn/env/lib/python2.7/site-packages/sqlalchemy/dialects/sqlite/pysqlite.py", line 339, in dbapi
    raise e
ImportError: /opt/mhn/env/lib/python2.7/site-packages/pysqlite2/_sqlite.so: undefined symbol: PyOS_mystrnicmp
```

In the Vagrantfile hashicorp/precise64 is used for the server base box.

**Comment by X4mp** [comment]
I also tried it with the ubuntu/trusty64 (14.04) base box and get the same ImportError.

**Comment by X4mp** [comment]
fixed it upgrading Python to 2.7.6
